packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetailsModal.tsx

```diff
@@ -105,7 +105,13 @@ const useBatchRows = (summaryItem: OutboundShipmentSummaryItem | null) => {
           const matchingInvoiceRow = Object.values(summaryItem.batches).find(
             ({ stockLineId }) => stockLineId === batch.id
           );
-          return { ...batch, quantity: matchingInvoiceRow?.numberOfPacks ?? 0 };
+          return {
+            ...batch,
+            quantity: matchingInvoiceRow?.numberOfPacks ?? 0,
+            availableNumberOfPacks:
+              batch.availableNumberOfPacks +
+              (matchingInvoiceRow?.numberOfPacks ?? 0),
```

Comment:
I'm not sure if this is right, need to confirm as the docs have:
![image](https://user-images.githubusercontent.com/35858975/141886420-63eb5c89-c97c-4c94-ab76-29e55b2c3390.png)

But I don't think that makes sense for all stock lines, so assuming that the reduction in the invoice is applied to the available quantity returned for the api for the stock line, then..

- The 'total available' includes reductions from any invoice lines.
- So, to get the real 'available' amount, we need to add on what is on the invoice line (if the invoice is > draft?)

But, this gets messy of course.

Available(A) = 100, Invoice line packs (B) = 50, Real available(C) = 150
1. Open dialog to edit: A=100, B=50, so we display C=150
2. Reduce invoice line from 50 to 20 (A = 100, B=20, (C) = 170)
3. Open the form again without saving, we display 100(A)+20(B)=120(C)

So, we would need to store an 'untouched number of packs' value to fix this.